src/server/game/Entities/Player/PlayerUpdates.cpp

```diff
@@ -0,0 +1,2311 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU
+ * GPL v2 license, you may redistribute it and/or modify it under version 2 of
+ * the License, or (at your option), any later version. Copyright (C) 2008-2016
+ * TrinityCore <http://www.trinitycore.org/> Copyright (C) 2005-2009 MaNGOS
+ * <http://getmangos.com/>
+ */
+
+#include "BattlefieldMgr.h"
+#include "CellImpl.h"
+#include "Channel.h"
+#include "ChannelMgr.h"
+#include "Formulas.h"
+#include "GridNotifiers.h"
+#include "GridNotifiersImpl.h"
+#include "Group.h"
+#include "Guild.h"
+#include "InstanceScript.h"
+#include "Language.h"
+#include "OutdoorPvPMgr.h"
+#include "Pet.h"
+#include "Player.h"
+#include "SavingSystem.h"
+#include "ScriptMgr.h"
+#include "SkillDiscovery.h"
+#include "SpellAuraEffects.h"
+#include "UpdateFieldFlags.h"
+#include "Vehicle.h"
+#include "WeatherMgr.h"
+
+// Zone Interval should be 1 second
+#define ZONE_UPDATE_INTERVAL (1 * IN_MILLISECONDS)
+
+#define CINEMATIC_UPDATEDIFF 500
+#define CINEMATIC_LOOKAHEAD  (2 * IN_MILLISECONDS)
+
+void Player::Update(uint32 p_time)
+{
+  if(!IsInWorld())
+    return;
+
```

Comment:
Use 4 whitespace for all

Reply:
Will be added to https://github.com/azerothcore/azerothcore-wotlk/pull/6848 if i can find a option for that

Reply:
so its supposed to be?
```cpp

    if(!IsInWorld())
    return;
```

Reply:
no.
```cpp
    if(!IsInWorld())
        return;

Reply:
That is the original code, how is this related to this PR?

Reply:
Because they are easy fixes, especially when pointed out at the exact line.

Reply:
Well i guess this pr is gonna wait then because i cant fix them as of now

Reply:
You already did, you dont need to fix them all in this PR. Only this one as it was pointed out

Reply:
Well this isnt usually how PR reviews are handled but I'll take it as that.

Reply:
I mean you are just moving code, I won't force you to also refactor it to be okay with the code standards set today.

Reply:
Yeah im just saying it might take a while until i can fix those currently, might or might not

Reply:
@Kitzunu By the way there are 36 instances of the same if statement and some like in `object.cpp` are the same, but i will change it in playerupdates.cpp now, i guess those files should also get refactored

Reply:
Alright this should be fixed

Reply:
There is a lot of old bad code, but that is for another day.

The c++ standards we use was recently put in place